packages/client/src/runner.ts

```diff
@@ -67,24 +70,42 @@ async function main(
     (global as any).__codechecks_client = _client;
 
     await executeCodechecksFile(codecheckFile);
+
+    successCodechecks += _client.countSuccesses();
+    failureCodechecks += _client.countFailures();
   }
 
   const finishTime = new Date().getTime();
   const deltaTime = finishTime - startTime;
 
-  logger.log(`All done in ${bold(ms(deltaTime))}!`);
+  const result: string = [
+    failureCodechecks > 0 ? bold(red(`${failureCodechecks} failed`)) : false,
+    successCodechecks > 0 ? bold(green(`${successCodechecks} succeeded`)) : false,
+    `${failureCodechecks + successCodechecks} total`,
+  ]
+    .filter(Boolean)
+    .join(", ");
+
+  logger.log(`${bold("Checks:")} ${result}`);
+  logger.log(`${bold("Time:")}   ${ms(deltaTime)}`);
+
+  if (sharedExecutionCtx.isWithExitStatus && failureCodechecks > 0) {
```

Comment:
I believe here we could access `isWithExitStatus` directly from args.

Reply:
I'm hesitant to use this directly from args. I don't want to use it directly to avoid refactorings when the same information can not only be set via process args but also configured in codechecks.yml or using an environment variable. These are fairly plausible ways to influence the way CC can be run (in the future).

I can come up with the abstraction layer for config instead.

Reply:
> I can come up with the abstraction layer for config instead.

yes this seems like a way to go 👍